### ISSUE TITLE
Events not highlighting

### ISSUE THREAD

**Issue by krsyoung** [GeoffZhu/vue-event-calendar#40]
First, this is an excellent Vue component, thanks for all of the hard work!

I'm having a problem where the events aren't being highlighted in the calendar.  Wondering if I'm somehow doing something silly with the date format (although I have tried many combinations without success).  Here is a screenshot:

![localhost-8080- iphone 6](https://user-images.githubusercontent.com/5449541/28997599-919267bc-79e6-11e7-8e24-6e8311580e41.png)

Not sure if this is a part of the problem but it seems weird that the date at the top is "MM/YYYY" but the format in the event is "YYYY/MM/DD".   I also tried using a date format of "YYYY/M/D" (to see if https://github.com/GeoffZhu/vue-event-calendar/issues/1 resurfaced), but it didn't help.

Here is the relevant code:

```
# Calendar.vue
<template>
  <vue-event-calendar
    :events="demoEvents"
    @month-changed="handleMonthChanged"
    @day-changed="handleDayChanged"
  >
    <template scope="props">
      <div v-for="(event, index) in props.showEvents" class="event-item">
        <!-- In here do whatever you want, make you owner event template -->
        {{event}}
      </div>
    </template>
  </vue-event-calendar>
</template>

<script>
export default {
  data () {
    return {
      claims: [],
      demoEvents: [{
        // id: 5,
        date: '2016/12/15',
        title: 'Foo',
        desc: 'longlonglong description'
      },
      {
        // id: 6,
        date: '2016/11/12',
        title: 'Bar'
      }]
    }
  },
  methods: {
    handleMonthChanged (data) {
      console.log('month-changed', data)
    },
    handleDayChanged (data) {
      console.log('month-changed', data)
    }
  }
}
</script>

```

**Comment by krsyoung** [comment]
Well I'm confused now.  I wrote an update because I thought the issue was I needed a date formatted as `YYYY/M/DD` (the single `M`), however then I did / undid that and it didn't work.  

Then I noticed I had the `title` missing, and when I added that and it started to work.  

Then I reverted the date format back to `YYYY/MM/DD` and it still works.  So I'm confused but it seems to work!  The circles are a little off but that is my problem :-) 

![localhost-8080- iphone 6 1](https://user-images.githubusercontent.com/5449541/28997823-fa13afbc-79eb-11e7-8ded-32f146091cf7.png)
